packages/jest-runtime/src/script_transformer.js

```diff
@@ -168,7 +169,9 @@ export default class ScriptTransformer {
           },
         ],
       ],
-    }).code;
+    });
+
+    return result ? result.code : content;
```

Comment:
As noted in https://github.com/facebook/jest/pull/6949#discussion_r216516336, I was actually mistaken about needing to handle `null` values here.

Reply:
Turns out this is required. Some tests broke after not checking for null.